pkg/operator/ceph/operator.go

```diff
@@ -156,7 +156,8 @@ func (o *Operator) Run() error {
 	}
 
 	// Start the controller-runtime Manager.
-	go o.startManager(namespaceToWatch, stopChan)
+	mgrErrorChan := make(chan struct{})
```

Comment:
What about an `error` type instead of a generic struct? So that when there is an error, we don't logger but assign it to the channel and then print the `error` in the case?

Reply:
`logger` here means the four log messages, just before `mgrErrorCh <- struct{}{}`, in pkg/operator/ceph/cr_manager.go?

Reply:
Yes, exactly, expect that `logger.Errorf(")` is not needed and `fmt.Errorf` is `error.Wrapf`.

Reply:
Thanks, fixed.